standards/alerting.md

```diff
@@ -1,3 +1,45 @@
-# Alerting & Notifications
+# Monitoring & Alarms
 
-This is a placeholder for linking until the standard has been drafted.
\ No newline at end of file
+All components MUST have an alarm set up.
```

Comment:
The word 'components' can have different meanings (e.g. React components, NPM components) should we tailor this to Services such as Lambdas, Back-end Services, API's?